doc/build-unix.md

```diff
@@ -69,7 +69,7 @@ Build requirements:
 
 Options when installing required Boost library files:
 
-1. On at least Ubuntu 14.04+ and Debian 7+ there are generic names for the
+1. On at least Ubuntu 14.04+ ( > kernel 3.13 )  and Debian 7+( > kernel v3.2.41)  there are generic names for the
```

Comment:
Where do you get these kernel recommendations from? Where does bitcoin core require a certain kernel?

I don't think they belong here at least: this tells what boost packages to install based on the distribution, the boost version has nothing to do with kernel version.